bot.py

```diff
@@ -113,18 +113,43 @@ def help_message() -> str:
     """
     global config
 
+    commands = ["howmany", "submit", "vote", "status", "myresults"]
+    admin_commands = ["getentryplacements", "postentries", "postentriespreview", "manage"]
+
     msg = ("Hey there! I'm 8Bot-- My job is to help you participate in "
            "the 8Bit Music Theory Discord Weekly Composition Competition.\n")
 
     if compo.get_week(True)["submissionsOpen"]:
         msg += "Submissions for this week's prompt are currently open.\n"
         msg += "If you'd like to submit an entry, DM me the command `" + \
-            config["command_prefix"][0] + "submit`, and I'll give you "
-        msg += "a secret link to a personal submission form."
+            client.command_prefix[0] + "submit`, and I'll give you "
+        msg += "a secret link to a personal submission form. \n"
     else:
         msg += "Submissions for this week's prompt are now closed.\n"
         msg += ("To see the already submitted entries for this week, "
-                "head on over to " + config["url_prefix"])
+                "head on over to " + config["url_prefix"]) + "\n"
+
+    if not full:
+        msg += "Send `" + client.command_prefix[0] + "help` to see all available "
+        msg += "commands."
+    else:
+        msg += "\nI understand the following commands: \n"
+
+        for command in commands:
+            msg += "`" + client.command_prefix[0] + command + "`: "
+            msg += client.get_command(command).short_doc + "\n"
+
+        if is_admin:
+            msg += "\nAlso since you're an admin, here are some secret commands: \n"
+
+            for command in admin_commands:
+                msg += "`" + client.command_prefix[0] + command + "`: "
+                msg += client.get_command(command).short_doc + "\n"
+
+        if len(client.command_prefix) > 1:
+            msg += "\n"
+            msg += "Besides `" + client.command_prefix[0] + "` I understand the"
```

Comment:
```suggestion
            msg += "Besides `" + client.command_prefix[0] + "` I understand the "
```
Space: the final frontier